docker-compose.yml

```diff
@@ -20,11 +33,9 @@ services:
     build:
       context: ui
       dockerfile: Dockerfile
-    volumes:
-      - /frontend:/app
     network_mode: "host"
```

Comment:
Now if we have separate docker-compose for dev and production, network_mode host should be removed and we should replace build context with deployed image. I don't think it's convenient that user has to build images.

Reply:
We can do these changes in docker-compose in a future PR.